.eslintrc

```diff
@@ -7,6 +7,9 @@
     "vars-on-top": 0,
     "no-use-before-define": 0,
     "space-before-function-paren": 0,
-    "max-len": [1, 80, 4, {"ignoreComments": true, "ignoreUrls": true}]
+    "max-len": [1, 80, 4, {"ignoreComments": true, "ignoreUrls": true}],
+    "no-param-reassign": 0,
+    "quote-props": 0,
+    "wrap-iife": [2, "inside"]
```

Comment:
Just asking. Is there some reason for this or is it just a matter of preference?